src/main/java/com/mongodb/jdbc/MongoSQLDatabaseMetaData.java

```diff
@@ -1118,21 +1122,21 @@ public ResultSet getTypeInfo() throws SQLException {
                         new BsonElement(TYPE_NAME, new BsonString(BSON_BINDATA.getBsonName())),
                         new BsonElement(DATA_TYPE, new BsonInt32(Types.BINARY)),
                         new BsonElement(PRECISION, asBsonIntOrNull(BSON_BINDATA.getPrecision())),
-                        new BsonElement(LITERAL_PREFIX, n),
-                        new BsonElement(LITERAL_SUFFIX, n),
-                        new BsonElement(CREATE_PARAMS, n),
+                        new BsonElement(LITERAL_PREFIX, BSON_NULL_VALUE),
```

Comment:
I noticed that other places in the file, we use `BsonNull.VALUE`, which is a class-level constant defined _in_ the java driver. They also have `BsonBoolean.FALSE`. Maybe we should defer to those constants instead of creating our own `BSON_NULL_VALUE` and `BSON_FALSE_VALUE` constants in this class! That we we're just making things like `BSON_ZERO_VALUE` that don't already exist somewhere.